internal/cmd/local/command_kafka.go

```diff
@@ -0,0 +1,23 @@
+package local
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/internal/pkg/cmd"
+)
+
+func NewKafkaCommand(prerunner cmd.PreRunner) *cobra.Command {
+	c := NewLocalCommand(
+		&cobra.Command{
+			Use:   "kafka",
+			Short: "Run Kafka related commands",
+			Long:  `----`,
```

Comment:
I take it this PR is not ready for review yet?

Reply:
no, it's just a progress tracking for the cp team

Reply:
Feel free to mark as "Draft" in that case!